libs/external-db-mysql/src/sql_filter_transformer.spec.ts

```diff
@@ -407,18 +408,36 @@ describe('Sql Parser', () => {
         })
 
     })
+    
+    interface Context {
+        fieldName: any
+        fieldValue: any
+        fieldListValue: any
+        anotherFieldName: any
+        moreFieldName: any
+        filter: any
+        anotherFilter: any
+        anotherValue: any
+        moreValue: any
+    }
 
-    const ctx = {
+    const ctx: Context = {
         fieldName: Uninitialized,
         fieldValue: Uninitialized,
         fieldListValue: Uninitialized,
         anotherFieldName: Uninitialized,
         moreFieldName: Uninitialized,
         filter: Uninitialized,
         anotherFilter: Uninitialized,
+        anotherValue: Uninitialized,
+        moreValue: Uninitialized
+    }
+
+    interface Enviorment {
+        filterParser: any
```

Comment:
why is this an `any` type?

Reply:
Because I made interface for filter parser only for the functions that used in data provider and not to all the class.